apps/climatemappedafrica/src/components/HURUmap/Panel/DesktopPanel/PanelItem.js

```diff
@@ -3,7 +3,7 @@ import React from "react";
 
 import RichData from "./RichData";
 
-function PanelItem({ item, ...props }) {
+function PanelItem({ item = undefined, ...props }) {
```

Comment:
```suggestion
function PanelItem({ item, ...props }) {
```

Isn't `undefined` the default in JS/TS?